src/Clay/Selector.hs

```diff
@@ -133,6 +133,12 @@ attr = Refinement . pure . Attr
 (|=) :: Text -> Text -> Refinement
 (|=) a = Refinement . pure . AttrHyph a
 
+-- | Convert a Refinement to a Selector. This keeps there from being
```

Comment:
Can you add an example where it would go wrong, and an example how your approach fixes it?